static/src/javascripts/projects/commercial/modules/header-bidding/prebid/prebid.ts

```diff
@@ -328,6 +329,8 @@ const initialise = (window: Window, framework = 'tcfv2'): void => {
 		 * set here when adjusting the slot size.
 		 * */
 		advert.hasPrebidSize = true;
+
+		if (data.bidderCode === 'improvedigital') captureCommercialMetrics();
```

Comment:
We’ll capture all of Improve Digital’s bids, but that has been deemed acceptable.

To filter out, we’d need to do something 

```ts
window.pbjs.onEvent(
	'bidWon',
	(data) => {
		if (Array.isArray(data.params) && isObject(data.params[0])) {
			if (
				String(getImproveSkinPlacementId()) ===
				String(data.params[0].placementId)
			)
				captureCommercialMetrics();
		}
	},
	'improvedigital',
);
```